### PR TITLE
Scrutinizer Auto-Fixes

### PR DESCRIPTION
**Pull request by scrutinizer-auto-fixer**
@jmontoyaa requested this pull request.

It consists of patches automatically generated for this project on Scrutinizer:
https://scrutinizer-ci.com/g/chamilo/chamilo-lms/

### CODE REVIEW

File: main/admin/access_url_add_usergroup_to_url.php [hunk 1]
```diff
@@ -87,7 +87,7 @@
                   <?php
                     echo Display::get_alphabet_options($firstLetterUserGroup);
                     echo Display::get_numeric_options(0, 9, $firstLetterUserGroup);
-                  ?>
+                    ?>
```
Comment (ywarnier) [inline main/admin/access_url_add_usergroup_to_url.php]: !? That's weird... why would it recommend to end the php tag at this level of indentation?

File: main/cron/document/index_all_docs.php [hunk 2]
```diff
@@ -40,30 +40,30 @@
...
     while ($row = Database::fetch_array($res)) {
-      $doc_path = api_get_path(SYS_COURSE_PATH).$course_dir.$row['path'];
-      $extensions = preg_split("/[\/\\.]/", $doc_path);
-      $doc_ext = strtolower($extensions[count($extensions) - 1]);
-      if (in_array($doc_ext, $allowed_extensions) && !is_dir($doc_path)) {
+        $doc_path = api_get_path(SYS_COURSE_PATH).$course_dir.$row['path'];
+        $extensions = preg_split("/[\/\\.]/", $doc_path);
+        $doc_ext = strtolower($extensions[count($extensions) - 1]);
+        if (in_array($doc_ext, $allowed_extensions) && !is_dir($doc_path)) {
         $doc_mime = mime_content_type($doc_path);
```
Comment (ywarnier) [inline main/cron/document/index_all_docs.php]: And here it's weird that it wouldn't catch the missing indentation...

File: main/document/slideshowoptions.php [hunk 3]
```diff
@@ -99,35 +99,35 @@ function disableresizing() { //v2.0
...
             <label class="col-sm-1 control-label"><?php echo get_lang('Width'); ?></label>
             <div class="col-sm-3">
                 <input class="form-control" name="width" type="text" id="width" <?php
-		if ($image_resizing == 'resizing') {
-			echo ' value="'.$width.'"';
-			echo ' class="enabled_input"';
+        if ($image_resizing == 'resizing') {
+            echo ' value="'.$width.'"';
+            echo ' class="enabled_input"';
                     } else {
```
Comment (ywarnier) [inline main/document/slideshowoptions.php]: A weird spacing improvement...

File: main/admin/access_url_add_usergroup_to_url.php (hunk 1 above)
Comment (jmontoyaa) [inline main/admin/access_url_add_usergroup_to_url.php]: I think the mix between html and php, breaks phpstorm and also scrutinizer

File: main/dropbox/dropbox_functions.inc.php [hunk 4]
```diff
@@ -84,11 +84,11 @@ function handle_multiple_actions()
     if (strstr($_POST['action'], 'move_')) {
         // check move_received_n or move_sent_n command
         if (strstr($_POST['action'], 'received')) {
-              $part = 'received';
-              $to_cat_id = str_replace('move_received_', '', $_POST['action']);
+                $part = 'received';
+                $to_cat_id = str_replace('move_received_', '', $_POST['action']);
```
Comment (ywarnier) [inline main/dropbox/dropbox_functions.inc.php]: This is not correct either. It replaces 2 spaces with 4 spaces. That's OK, but then it misses the fact that there are now 8 spaces of indentation with the upper level...

File: main/document/slideshowoptions.php (hunk 3 above)
Comment (jmontoyaa) [inline main/document/slideshowoptions.php]: Same html+php behaviour

**Comment by jmontoyaa** [comment]
I run again scrutinizer with new rules.

**Pull request merged**